frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu.dart

```diff
@@ -104,6 +107,35 @@ class HomeMenu extends StatelessWidget {
     );
   }
 
+  Widget _renderFavorites(BuildContext context) {
+    return BlocProvider(
+      create: (context) =>
+          getIt<FavoriteBloc>()..add(const FavoriteEvent.initial()),
+      child: BlocConsumer<FavoriteBloc, FavoriteState>(
+        listener: (context, state) {
+          Log.warn("Favorite state changed $state");
+        },
+        builder: (context, state) {
+          return Visibility(
+            visible: true,
+            child: ExpandableTheme(
+              data: ExpandableThemeData(
+                useInkWell: true,
+                animationDuration: Durations.medium,
+              ),
+              child: ScrollConfiguration(
+                behavior: const ScrollBehavior().copyWith(scrollbars: false),
+                child: Column(
+                  children: [...state.objects.map((e) => Text(e.id)).toList()],
```

Comment:
You can remove the spread operator and the brackets `children: state.objects.map().toList(),`